guides/v2.1/extension-dev-guide/cache/partial-caching/database-caching.md

```diff
@@ -13,29 +13,30 @@ redirect_from:
   -  /guides/v2.2/config-guide/database/database.html
 ---
 
-This topic discusses how to use the Magento 2 database for caching. After you complete these tasks, cached objects are stored in the `cache` and `cache_tag` Magento 2 database tables. Nothing is stored in `var/cache` or `var/page_cache`.
+## Overview of database caching {#mage-cache-db-over}
```

Comment:
This heading is unnecessary. I know that there are several other topics on devdocs that use an introductory heading like this, but it's not necessary.

Reply:
Fixed